docs/releasing/README.md

```diff
@@ -25,16 +21,9 @@ export VERSION=v0.15.0
     git checkout -b "releases/$VERSION_RC"
     ```
 
-1. Update the [RELEASES.md](../../RELEASES.md) file with the new release version `$VERSION`.
+1. Update the [RELEASES.md](../../RELEASES.md) file by renaming the "Pending" section to the new release version `$VERSION` and creating a new "Pending" section at the top.
 1. Modify the [plugin/evm/version.go](../../plugin/evm/version.go) `Version` global string variable and set it to the desired `$VERSION`.
-1. Ensure the AvalancheGo version used in [go.mod](../../go.mod) is [its last release](https://github.com/ava-labs/avalanchego/releases). If not, upgrade it with, for example:
-
-    ```bash
-    go get github.com/ava-labs/avalanchego@v1.13.0
-    go mod tidy
-    ```
-
-    And fix any errors that may arise from the upgrade. If it requires significant changes, you may want to create a separate PR for the upgrade and wait for it to be merged before continuing with this procedure.
+1. Because AvalancheGo and coreth depend on each other, and that we create releases of AvalancheGo before coreth, you can use a recent commit hash or recent release candidate of AvalancheGo in your `go.mod` file.
```

Comment:
maybe we can add a note that coreth releases should be coordinated with avalanchego releases

Reply:
Good point! Done!